### DISCUSSION TITLE
Resource based scheduling issues

### DISCUSSION THREAD

**Discussion by theskysea** [frePPLe/frepple#542] [Q&A]
Setup one item can be operated by three resources.
I don't want all three of them open at the same time to operate.
Because our resources are expensive to turn on once. Therefore, I want to open one resource production, if the order is delayed, then open a second resource, if the delay is still delayed, then open a third resource.
How to set this up in the frepple.

thanks.

**Comment by jdetaeye** [comment]
If you configure your model with a) search mode "priority" among resource or operations and b) a "maxearly" field on the resources that is sufficiently big, then you'll get exactly the behavior you describe.

The "priority" search mode is the default and the "maxearly" field is 100 days by default. So, in many situations you'll get this behavior out of the box already.
